mplc/doc/documentation.md

```diff
@@ -544,6 +544,23 @@ These are not mandatory, by default the local dataset will not be split.
 
 > Note: currently, the local validation and test set are not used, but they are available for further developments of multi-partner learning and contributivity measurement approaches.
 
+## Run the tests
+
+You will first need to install the dev dependencies with `pip3 install -r dev-requirements.txt` and then run:
+
+```sh
+# Run unit tests
+pytest -vv tests/unit_tests.py
+# or 
+python3 -m pytest -vv tests/unit_tests.py
+
+# Fall in debugger breakpoint in case of error
+pytest -vv tests/unit_tests.py --pdb
+
+# Run tests and produce a coverage measure. This will output a .coverage file
+pytest -vv --cov=mplc tests/unit_tests.py
+```
+
```

Comment:
I am not sure, why don't put these explanations in the CONTRIBUTING.md file ?

Reply:
You mean removing this from the documentation.md?

Reply:
I have added this to the contributing doc